src/EnergyPlus/GeneralRoutines.cc

```diff
@@ -1139,74 +1089,50 @@ void PassiveGapNusseltNumber(Real64 const AspRat, // Aspect Ratio of Gap height
     // REFERENCES:
     // Window5 source code; ISO 15099
```

Comment:
Definitely something that belongs in "GeneralRoutines". I use this all the time.

Reply:
This is so general we talk about it at family dinner.